src/v2/Apps/Consign/OfferDetail/Components/ResponseForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react"
+import { Form, Formik } from "formik"
+import { BorderedRadio, Button, RadioGroup, Text } from "@artsy/palette"
+import { useSystemContext } from "v2/Artsy"
+import { CreateOfferResponse } from "./CreateOfferResponse"
+import { ResponseForm_offer } from "v2/__generated__/ResponseForm_offer.graphql"
+import { createFragmentContainer, graphql } from "react-relay"
+import { CreateOfferResponseMutationInput } from "v2/__generated__/CreateOfferResponseMutation.graphql"
+
+interface ResponseFormProps {
+  offer: ResponseForm_offer
+}
+
+type ResponseFormValues = Pick<
+  CreateOfferResponseMutationInput,
+  "intendedState"
+>
+
+const ResponseForm: React.FC<ResponseFormProps> = ({ offer }) => {
+  const { relayEnvironment } = useSystemContext()
+  const [responded, setResponded] = useState(false)
+
+  if (responded) {
+    return <Text variant="title">Thank you! We'll be in touch soon.</Text>
+  }
+
+  return (
+    <Formik
+      initialValues={{
+        intendedState: "ACCEPTED",
+      }}
+      onSubmit={async (values: ResponseFormValues, actions) => {
+        await CreateOfferResponse(relayEnvironment, offer.id, values)
+        setResponded(true)
```

Comment:
I think I should probably put a `catch` in here, though Formik handles failed submissions well enough that you can still use the form afterward.